control/ping.yaml

```diff
@@ -0,0 +1,93 @@
+openapi: 3.1.0
+
+info:
+  title: Ping models
+  description: >-
+    All ping schemas
+  version: ^0.0.0
+
+components:
+  schemas:
+    Ping:
+      type: object
+      properties:
+        choice:
+          description: ipv4 or ipv6 ping
+          type: string
+          enum: [ipv4, ipv6]
+          default: ipv4
+        ipv4:
+          $ref: '#/components/schemas/Ping.Ipv4'
+        ipv6:
+          $ref: '#/components/schemas/Ping.Ipv6'
+
+    Ping.Ipv4:
+      type: object
+      properties:
+        src_name:
+          description: >-
+            The name of the source from where ping is initiated.
+          type: string
+          x-constraint:
+          - '/components/schemas/Device.Ipv4Base/properties/name'
+        dst_ips:
+          description: >-
+              Array of destination ipv4 addresses
+          type: array
+          items:
+            description: >-
+              destination address
+            type: string
+            format: ipv4
+    
+    Ping.Ipv6:
+      type: object
+      properties:
+        src_name:
+          description: >-
+            The name of source to send ping.
```

Comment:
Make this description similar to the one for ipv4 src_name

Reply:
updated